faq/databases-for-redis.mdx

```diff
@@ -0,0 +1,74 @@
+One of the most common ways to implement cache is storing frequently accessed data in Redis™ (therefore in memory), and serving your application's request from there, should data be available, instead of from your traditional database. If data is not available in memory, it can always be retrieved from the primary database. Below is a simplified schema explaining this process . 
+
+## How can I access my Redis once it is provisioned?
+
+You can connect to your Redis™  using the IP address.es and port.s available in the details page of your Database for Redis™ Instance next to the title **Endpoints**.
+- There are as many `IP:port` as nodes in your cluster (one if standalone).
+- If you are in cluster mode (3 nodes or more), you have to use a cluster specific connector able to connect to multiple endpoints.
+
+ADD SCREENSHOT.
+
+## What is the Private Networks feature for Database Instances?
+
+[Private Networks](https://www.scaleway.com/en/docs/network/vpc/quickstart/) allows you to configure your own Layer-2 network with IPv4 addresses on Database for Redis™ Instances. 
+You can therefore enhance the security of your system architecture by isolating it from the Internet. It also increases the security of your databases, since Instances in your Private Network can directly be connected to your Redis™ Instance, without passing through the public Internet.
+- In standalone mode, you can create new Redis™ Instances to attach to your Private Network, or attach existing Database Instances by adding Private Networks endpoints to them. 
+- In standalone mode, you can attach Private Networks to your Redis™ Instances at the creation level or add a Private Networks endpoint to your existing Redis™ Instance from the console.
+
+<Message type="note">
+
+Bear in mind that in cluster mode, Private Network is only available at the creation of the cluster Instance and cannot be modified afterwards. If you want to switch back to a public network or update your IP adresses, you have to create a new Redis Instance. 
+
+</Message>
+
+## How many Private Networks can I attach?
+ 
+Currently, you can connect one Private Network to each Database for Redis Instance in the Scaleway Console. For Standalone Instances you can attach multiple Private Networks via the CLI or the API only.
+
+## Can I use any IP Range for my Private Network?
+ 
+For now, only the [RFC1918](https://datatracker.ietf.org/doc/html/rfc1918) IP subnets are accepted as a Database for Redis Instance virtual IPv4.
+
+## How is the Redis Database's failover IP managed?
+
+The Redis controller is aware of the cluster topology and routes requests automatically to a healthy node. In case of unavailability of one of the cluster nodes, no downtime is to be expected.
```

Comment:
In cluster mode (3 nodes and above), the RedisTM controller is aware of the cluster topology and routes requests automatically to a healthy node. In case of unavailability of one of the cluster nodes, no downtime is to be expected.